Excercise/Exercise3.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>HTML Exercise 3</title>
+</head>
+<body>
+	<h1>Logos</h1>
+	<img src="logos.png" alt="logos">
+	<br>
+	<a href="jackinthebox.html" target="_blank">Jack in the box</a>
```

Comment:
Please create clickable areas inside the image, that image is a collection of the individual logo. try to use image map to complete the exercise.